content/en/synthetics/browser_tests/_index.md

```diff
@@ -131,6 +155,10 @@ By default, only users with the [Datadog Admin and Datadog Standard roles][14] c
 
 If you have access to the [custom role feature][15], add your user to any custom role that includes `synthetics_read` and `synthetics_write` permissions.
 
+### Select roles
+
+Restrict access to your browser test for yourself and custom roles by clicking the roles in the **Everyone in my org** dropdown menu.
```

Comment:
we should reuse the Restrict access already done for global var 
https://docs.datadoghq.com/synthetics/settings/?tab=specifyvalue#restrict-access

Reply:
Good point, I'll make a new screenshot!